go.mod

```diff
@@ -7,9 +7,11 @@ require (
 	github.com/aws/aws-sdk-go-v2/config v1.25.11
 	github.com/aws/aws-sdk-go-v2/service/applicationautoscaling v1.25.5
 	github.com/aws/aws-sdk-go-v2/service/cloudwatch v1.32.0
+	github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs v1.30.1
```

Comment:
This was added via `go mod tidy` and is unrelated to the functionality added in this PR.